packages/radio-player/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@internetarchive/radio-player",
+  "version": "0.0.0",
+  "description": "A Radio Player LitElement.",
+  "license": "AGPL-3.0-only",
+  "main": "index.js",
+  "module": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "lib/**/*",
+    "!lib/demo",
```

Comment:
question: is the `!` intentional?

Reply:
Yeah. This is the list of files that we want to package into the npm module when we upload it to npmjs.io and we want to ignore the demo directory.